tests/unit/TableTest.php

```diff
@@ -327,4 +273,28 @@ public function testDeleteColumn()
 
         $this->assertNull($r);
     }
+
+    protected function createTestTable(string $tableName,bool $hasAge = true):Morebec\YDB\Table
+    {
+        $schema = $this->createTestTableSchema($hasAge);
+
+        return $this->database->createTable(
+            new TableSchema($tableName,$schema)
+        );
+    }
+
+    protected function createTestTableSchema(bool $hasAge)
```

Comment:
Since Schemas and Table are always created together I don't really see a point to have a separate function for this.
I would prefer it, if you would put the schema with the table